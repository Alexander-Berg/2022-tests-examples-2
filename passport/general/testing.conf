[tvm_keyring]
/passport_api = tvm_keyring/testing.tpl:sec-01cj4dhv5ytg0agc54db6d3bb6[testing]

[statbox_push_client]
/push-client-backend.yaml.tvm = sec-01cj4dhv5ytg0agc54db6d3bb6[testing]

[backend_secrets]
/settings.py = python_settings.tpl:sec-01cjdcda3vy0q5vg05pk59cycf->passport_settings,sec-01ezsb0q3jsh6tcm0tqrh50gqk->dbscripts_settings

[historydb]
/. = sec-01d0xvxrrab3z13fhc2rdmnbbh[*.key]
