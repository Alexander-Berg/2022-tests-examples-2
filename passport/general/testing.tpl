{
  "client_id": 2000079,
  "client_secret": "{{ testing }}"
}
