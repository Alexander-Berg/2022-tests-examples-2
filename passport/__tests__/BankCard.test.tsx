import React from 'react';

import { BankCard, BankCardPreview } from '..';
import { createClientRender, screen } from '../../../libs/testing';
import { BankCardType, CardBank, CardSystem } from '../constants';

describe('BankCard', () => {
  const render = createClientRender();

  test('should set custom class name', () => {
    render(
      <BankCard
        className="custom-class"
        bankCardType={BankCardType.Default}
        bank={CardBank.Akbars}
        paymentSystem={CardSystem.AmericanExpress}
        cardNumber="4100"
        data-testid="element"
      />,
    );

    expect(screen.getByTestId('element')).toHaveClass('custom-class');
  });

  describe('Snapshots', () => {
    test('Default', () => {
      const result = render(
        <BankCard
          bank={CardBank.ATB}
          paymentSystem={CardSystem.DinersClub}
          bankCardType={BankCardType.Default}
          cardNumber="** 4100"
        />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('WithLabel', () => {
      const result = render(
        <BankCard
          bank={CardBank.ATB}
          paymentSystem={CardSystem.DinersClub}
          bankCardType={BankCardType.Default}
          cardNumber="** 4100"
          label="label"
        />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('WithMoney', () => {
      const result = render(
        <BankCard
          bank={CardBank.ATB}
          paymentSystem={CardSystem.DinersClub}
          bankCardType={BankCardType.Default}
          cardNumber="** 4100"
          money="100 ₽"
        />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('WithLabelMoney', () => {
      const result = render(
        <BankCard
          bank={CardBank.ATB}
          paymentSystem={CardSystem.DinersClub}
          bankCardType={BankCardType.Default}
          cardNumber="** 4100"
          label="label"
          money="100 ₽"
        />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('FamilyOwner', () => {
      const result = render(
        <BankCard
          bank={CardBank.ATB}
          paymentSystem={CardSystem.DinersClub}
          bankCardType={BankCardType.Default}
          label="Семейная"
          cardNumber="** 4100"
        />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('FamilyMember', () => {
      const result = render(
        <BankCard bankCardType={BankCardType.FamilyMember} money="5000 ₽ на неделю" />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('Preview', () => {
      const result = render(
        <BankCardPreview bankCardType={BankCardType.Default} bank={CardBank.Alfa} />,
      );

      expect(result.asFragment()).toMatchSnapshot();
    });

    test('PreviewFamilyMember', () => {
      const result = render(<BankCardPreview bankCardType={BankCardType.FamilyMember} />);

      expect(result.asFragment()).toMatchSnapshot();
    });
  });
});
