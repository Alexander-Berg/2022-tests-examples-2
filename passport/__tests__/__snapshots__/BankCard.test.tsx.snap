// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BankCard Snapshots Default 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    style="background-color: rgb(34, 34, 34); color: rgb(255, 255, 255);"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="32"
          viewBox="0 0 32 32"
          width="32"
        >
          <path
            d="M24.048 4H7.993A3.99 3.99 0 0 0 4 7.986v16.028A3.99 3.99 0 0 0 7.993 28h16.013A3.99 3.99 0 0 0 28 24.014V7.986C28.041 5.786 26.252 4 24.048 4zm2.745 19.599c-.041.041-.125.083-.166.041a20.45 20.45 0 0 0-10.607-2.948A20.45 20.45 0 0 0 5.414 23.64c-.041.042-.125 0-.166-.041-.042-.083 0-.125.041-.166 2.829-2.16 5.241-4.9 7.113-8.139A27.644 27.644 0 0 0 15.896 5.08c0-.042.041-.125.124-.125.084 0 .125.042.125.125a27.642 27.642 0 0 0 3.494 10.214 27.38 27.38 0 0 0 7.113 8.097c.083.083.083.125.041.208z"
            fill="#FE5000"
          />
        </svg>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <h4
          data-family="false"
        >
          ** 4100
        </h4>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="24"
          viewBox="0 0 36 24"
          width="36"
        >
          <path
            d="M19.181 21c4.872.024 9.319-4.012 9.319-8.922 0-5.369-4.447-9.08-9.319-9.078h-4.193C10.058 2.998 6 6.71 6 12.078c0 4.91 4.058 8.946 8.988 8.922h4.193z"
            fill="#0079BE"
          />
          <path
            d="M15.008 3.744c-4.505.002-8.156 3.689-8.157 8.24.001 4.55 3.652 8.237 8.157 8.239 4.507-.002 8.158-3.689 8.159-8.239-.001-4.551-3.652-8.238-8.159-8.24zm-5.17 8.24a5.23 5.23 0 0 1 3.32-4.874v9.747a5.227 5.227 0 0 1-3.32-4.873zm7.02 4.875V7.11a5.227 5.227 0 0 1 3.322 4.875 5.228 5.228 0 0 1-3.321 4.875z"
            fill="#fff"
          />
          <path
            d="M20.06 21c5.197.024 9.94-4.012 9.94-8.922C30 6.71 25.257 2.998 20.06 3h-4.472C10.328 2.998 6 6.71 6 12.078c0 4.91 4.329 8.946 9.588 8.922h4.472z"
            fill="#0079BE"
          />
          <path
            d="M15 5a7.001 7.001 0 0 0 0 14 7 7 0 0 0 0-14zm-4.437 7a4.444 4.444 0 0 1 2.848-4.14v8.28A4.442 4.442 0 0 1 10.563 12zm6.024 4.142V7.86a4.442 4.442 0 0 1 2.85 4.14 4.443 4.443 0 0 1-2.85 4.142z"
            fill="#fff"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots FamilyMember 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <div
          class="UserPic"
        >
          <img
            class="UserPic-Image"
            src="https://avatars.mds.yandex.net/get-yapic/0/0-0/islands-middle"
            srcset="https://avatars.mds.yandex.net/get-yapic/0/0-0/islands-middle 1x, https://avatars.mds.yandex.net/get-yapic/0/0-0/islands-retina-middle 2x"
          />
        </div>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <h4
          data-family="true"
        >
          5000 ₽ на неделю
        </h4>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots FamilyOwner 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    style="background-color: rgb(34, 34, 34); color: rgb(255, 255, 255);"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="32"
          viewBox="0 0 32 32"
          width="32"
        >
          <path
            d="M24.048 4H7.993A3.99 3.99 0 0 0 4 7.986v16.028A3.99 3.99 0 0 0 7.993 28h16.013A3.99 3.99 0 0 0 28 24.014V7.986C28.041 5.786 26.252 4 24.048 4zm2.745 19.599c-.041.041-.125.083-.166.041a20.45 20.45 0 0 0-10.607-2.948A20.45 20.45 0 0 0 5.414 23.64c-.041.042-.125 0-.166-.041-.042-.083 0-.125.041-.166 2.829-2.16 5.241-4.9 7.113-8.139A27.644 27.644 0 0 0 15.896 5.08c0-.042.041-.125.124-.125.084 0 .125.042.125.125a27.642 27.642 0 0 0 3.494 10.214 27.38 27.38 0 0 0 7.113 8.097c.083.083.083.125.041.208z"
            fill="#FE5000"
          />
        </svg>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <span
          data-family="false"
        >
          Семейная
        </span>
        <h4
          data-family="false"
        >
          ** 4100
        </h4>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="24"
          viewBox="0 0 36 24"
          width="36"
        >
          <path
            d="M19.181 21c4.872.024 9.319-4.012 9.319-8.922 0-5.369-4.447-9.08-9.319-9.078h-4.193C10.058 2.998 6 6.71 6 12.078c0 4.91 4.058 8.946 8.988 8.922h4.193z"
            fill="#0079BE"
          />
          <path
            d="M15.008 3.744c-4.505.002-8.156 3.689-8.157 8.24.001 4.55 3.652 8.237 8.157 8.239 4.507-.002 8.158-3.689 8.159-8.239-.001-4.551-3.652-8.238-8.159-8.24zm-5.17 8.24a5.23 5.23 0 0 1 3.32-4.874v9.747a5.227 5.227 0 0 1-3.32-4.873zm7.02 4.875V7.11a5.227 5.227 0 0 1 3.322 4.875 5.228 5.228 0 0 1-3.321 4.875z"
            fill="#fff"
          />
          <path
            d="M20.06 21c5.197.024 9.94-4.012 9.94-8.922C30 6.71 25.257 2.998 20.06 3h-4.472C10.328 2.998 6 6.71 6 12.078c0 4.91 4.329 8.946 9.588 8.922h4.472z"
            fill="#0079BE"
          />
          <path
            d="M15 5a7.001 7.001 0 0 0 0 14 7 7 0 0 0 0-14zm-4.437 7a4.444 4.444 0 0 1 2.848-4.14v8.28A4.442 4.442 0 0 1 10.563 12zm6.024 4.142V7.86a4.442 4.442 0 0 1 2.85 4.14 4.443 4.443 0 0 1-2.85 4.142z"
            fill="#fff"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots Preview 1`] = `
<DocumentFragment>
  <div
    bank="alfa"
    class=""
    style="background-color: rgb(239, 49, 36);"
  >
    <svg
      aria-hidden="true"
      class="SvgIcon"
      fill="none"
      focusable="false"
      height="24"
      viewBox="0 0 32 32"
      width="24"
    >
      <path
        clip-rule="evenodd"
        d="M16 2c1.98 0 2.603 1.366 3.124 2.733.22.577 5.363 14.138 5.516 14.545.044.112.015.3-.166.366l-1.694.633-1.308.488a.293.293 0 0 1-.37-.167l-.764-2.034v-.002l-.523-1.392h-7.619L10.9 20.599a.288.288 0 0 1-.368.168c-.116-.045-1.632-.611-2.466-.923l-.533-.199a.284.284 0 0 1-.17-.366c.098-.265 1.515-4 2.89-7.623v-.004c1.268-3.341 2.498-6.585 2.623-6.918C13.376 3.387 14.023 2 16.001 2zM6.957 26.263a.298.298 0 0 0-.206.085.276.276 0 0 0-.083.2v3.168c0 .158.128.284.289.284h18.09a.296.296 0 0 0 .205-.085.274.274 0 0 0 .082-.2v-3.168a.274.274 0 0 0-.082-.199.296.296 0 0 0-.204-.085H6.956zm6.453-12.385 2.577-7.45h.032l2.585 7.45h-5.194z"
        fill="#000"
        fill-rule="evenodd"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots PreviewFamilyMember 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    <svg
      aria-hidden="true"
      class="SvgIcon"
      fill="none"
      focusable="false"
      height="40"
      type="heart"
      viewBox="0 0 40 40"
      width="40"
    >
      <path
        d="M20.3916 27.0939C20.1521 27.2561 19.8479 27.2561 19.6084 27.0939C18.0017 26.0058 12 21.6672 12 17.569C12 12.1901 18.28 11.3285 20 14.952C21.72 11.3285 28 12.1901 28 17.569C28 21.6672 21.9983 26.0058 20.3916 27.0939Z"
        fill="white"
      />
      <path
        d="M6 17.569C6 20.0354 6.88746 22.1375 7.79013 23.6734C8.70963 25.2379 9.86434 26.5882 10.9201 27.6636C13.0271 29.8098 15.3034 31.4249 16.244 32.0619C18.5154 33.6001 21.4845 33.6001 23.756 32.0619C24.6966 31.4249 26.9729 29.8098 29.0799 27.6636C30.1357 26.5882 31.2904 25.2379 32.2099 23.6734C33.1125 22.1375 34 20.0354 34 17.569C34 9.38634 26.0822 5.04521 20 7.26364C13.9178 5.04521 6 9.38634 6 17.569Z"
        stroke="white"
        stroke-linejoin="round"
        stroke-opacity="0.1"
        stroke-width="12"
      />
      <path
        d="M20.3916 27.0939C20.1521 27.2561 19.8479 27.2561 19.6084 27.0939C18.0017 26.0058 12 21.6672 12 17.569C12 12.1901 18.28 11.3285 20 14.952C21.72 11.3285 28 12.1901 28 17.569C28 21.6672 21.9983 26.0058 20.3916 27.0939Z"
        fill="white"
      />
      <path
        d="M7.5 17.569C7.5 19.675 8.25937 21.5114 9.08331 22.9133C9.91988 24.3367 10.986 25.5896 11.9905 26.6127C13.996 28.6556 16.1788 30.2061 17.0851 30.8199C18.8486 32.0141 21.1514 32.0141 22.9149 30.8199C23.8212 30.2061 26.004 28.6556 28.0095 26.6127C29.014 25.5896 30.0801 24.3367 30.9167 22.9133C31.7406 21.5114 32.5 19.675 32.5 17.569C32.5 10.1973 25.236 6.56215 20 8.91822C14.764 6.56215 7.5 10.1973 7.5 17.569Z"
        stroke="white"
        stroke-linejoin="round"
        stroke-opacity="0.1"
        stroke-width="9"
      />
      <path
        d="M20.3916 27.0939C20.1521 27.2561 19.8479 27.2561 19.6084 27.0939C18.0017 26.0058 12 21.6672 12 17.569C12 12.1901 18.28 11.3285 20 14.952C21.72 11.3285 28 12.1901 28 17.569C28 21.6672 21.9983 26.0058 20.3916 27.0939Z"
        fill="white"
      />
      <path
        d="M9 17.569C9 19.3145 9.63128 20.8853 10.3765 22.1533C11.1301 23.4355 12.1077 24.591 13.0609 25.5619C14.965 27.5015 17.0542 28.9874 17.9262 29.5779C19.1817 30.4281 20.8183 30.4281 22.0738 29.5779C22.9458 28.9874 25.035 27.5015 26.9391 25.5619C27.8923 24.591 28.8699 23.4355 29.6235 22.1533C30.3687 20.8853 31 19.3145 31 17.569C31 10.9826 24.3372 8.0962 20 10.6411C15.6628 8.0962 9 10.9826 9 17.569Z"
        stroke="white"
        stroke-linejoin="round"
        stroke-opacity="0.1"
        stroke-width="6"
      />
      <path
        d="M20.3916 27.0939C20.1521 27.2561 19.8479 27.2561 19.6084 27.0939C18.0017 26.0058 12 21.6672 12 17.569C12 12.1901 18.28 11.3285 20 14.952C21.72 11.3285 28 12.1901 28 17.569C28 21.6672 21.9983 26.0058 20.3916 27.0939Z"
        fill="white"
      />
      <path
        d="M10.5 17.569C10.5 18.954 11.0032 20.2592 11.6697 21.3932C12.3404 22.5344 13.2294 23.5924 14.1313 24.5111C15.934 26.3473 17.9296 27.7686 18.7673 28.3359C19.5148 28.8421 20.4852 28.8421 21.2327 28.3359C22.0704 27.7686 24.066 26.3473 25.8687 24.5111C26.7706 23.5924 27.6596 22.5344 28.3303 21.3932C28.9968 20.2592 29.5 18.954 29.5 17.569C29.5 11.7083 23.311 9.66474 20 12.5121C16.689 9.66474 10.5 11.7083 10.5 17.569Z"
        stroke="white"
        stroke-linejoin="round"
        stroke-opacity="0.2"
        stroke-width="3"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots WithLabel 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    style="background-color: rgb(34, 34, 34); color: rgb(255, 255, 255);"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="32"
          viewBox="0 0 32 32"
          width="32"
        >
          <path
            d="M24.048 4H7.993A3.99 3.99 0 0 0 4 7.986v16.028A3.99 3.99 0 0 0 7.993 28h16.013A3.99 3.99 0 0 0 28 24.014V7.986C28.041 5.786 26.252 4 24.048 4zm2.745 19.599c-.041.041-.125.083-.166.041a20.45 20.45 0 0 0-10.607-2.948A20.45 20.45 0 0 0 5.414 23.64c-.041.042-.125 0-.166-.041-.042-.083 0-.125.041-.166 2.829-2.16 5.241-4.9 7.113-8.139A27.644 27.644 0 0 0 15.896 5.08c0-.042.041-.125.124-.125.084 0 .125.042.125.125a27.642 27.642 0 0 0 3.494 10.214 27.38 27.38 0 0 0 7.113 8.097c.083.083.083.125.041.208z"
            fill="#FE5000"
          />
        </svg>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <span
          data-family="false"
        >
          label
        </span>
        <h4
          data-family="false"
        >
          ** 4100
        </h4>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="24"
          viewBox="0 0 36 24"
          width="36"
        >
          <path
            d="M19.181 21c4.872.024 9.319-4.012 9.319-8.922 0-5.369-4.447-9.08-9.319-9.078h-4.193C10.058 2.998 6 6.71 6 12.078c0 4.91 4.058 8.946 8.988 8.922h4.193z"
            fill="#0079BE"
          />
          <path
            d="M15.008 3.744c-4.505.002-8.156 3.689-8.157 8.24.001 4.55 3.652 8.237 8.157 8.239 4.507-.002 8.158-3.689 8.159-8.239-.001-4.551-3.652-8.238-8.159-8.24zm-5.17 8.24a5.23 5.23 0 0 1 3.32-4.874v9.747a5.227 5.227 0 0 1-3.32-4.873zm7.02 4.875V7.11a5.227 5.227 0 0 1 3.322 4.875 5.228 5.228 0 0 1-3.321 4.875z"
            fill="#fff"
          />
          <path
            d="M20.06 21c5.197.024 9.94-4.012 9.94-8.922C30 6.71 25.257 2.998 20.06 3h-4.472C10.328 2.998 6 6.71 6 12.078c0 4.91 4.329 8.946 9.588 8.922h4.472z"
            fill="#0079BE"
          />
          <path
            d="M15 5a7.001 7.001 0 0 0 0 14 7 7 0 0 0 0-14zm-4.437 7a4.444 4.444 0 0 1 2.848-4.14v8.28A4.442 4.442 0 0 1 10.563 12zm6.024 4.142V7.86a4.442 4.442 0 0 1 2.85 4.14 4.443 4.443 0 0 1-2.85 4.142z"
            fill="#fff"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots WithLabelMoney 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    style="background-color: rgb(34, 34, 34); color: rgb(255, 255, 255);"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="32"
          viewBox="0 0 32 32"
          width="32"
        >
          <path
            d="M24.048 4H7.993A3.99 3.99 0 0 0 4 7.986v16.028A3.99 3.99 0 0 0 7.993 28h16.013A3.99 3.99 0 0 0 28 24.014V7.986C28.041 5.786 26.252 4 24.048 4zm2.745 19.599c-.041.041-.125.083-.166.041a20.45 20.45 0 0 0-10.607-2.948A20.45 20.45 0 0 0 5.414 23.64c-.041.042-.125 0-.166-.041-.042-.083 0-.125.041-.166 2.829-2.16 5.241-4.9 7.113-8.139A27.644 27.644 0 0 0 15.896 5.08c0-.042.041-.125.124-.125.084 0 .125.042.125.125a27.642 27.642 0 0 0 3.494 10.214 27.38 27.38 0 0 0 7.113 8.097c.083.083.083.125.041.208z"
            fill="#FE5000"
          />
        </svg>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <span
          data-family="false"
        >
          label
        </span>
        <h4
          data-family="false"
        >
          100 ₽
        </h4>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="24"
          viewBox="0 0 36 24"
          width="36"
        >
          <path
            d="M19.181 21c4.872.024 9.319-4.012 9.319-8.922 0-5.369-4.447-9.08-9.319-9.078h-4.193C10.058 2.998 6 6.71 6 12.078c0 4.91 4.058 8.946 8.988 8.922h4.193z"
            fill="#0079BE"
          />
          <path
            d="M15.008 3.744c-4.505.002-8.156 3.689-8.157 8.24.001 4.55 3.652 8.237 8.157 8.239 4.507-.002 8.158-3.689 8.159-8.239-.001-4.551-3.652-8.238-8.159-8.24zm-5.17 8.24a5.23 5.23 0 0 1 3.32-4.874v9.747a5.227 5.227 0 0 1-3.32-4.873zm7.02 4.875V7.11a5.227 5.227 0 0 1 3.322 4.875 5.228 5.228 0 0 1-3.321 4.875z"
            fill="#fff"
          />
          <path
            d="M20.06 21c5.197.024 9.94-4.012 9.94-8.922C30 6.71 25.257 2.998 20.06 3h-4.472C10.328 2.998 6 6.71 6 12.078c0 4.91 4.329 8.946 9.588 8.922h4.472z"
            fill="#0079BE"
          />
          <path
            d="M15 5a7.001 7.001 0 0 0 0 14 7 7 0 0 0 0-14zm-4.437 7a4.444 4.444 0 0 1 2.848-4.14v8.28A4.442 4.442 0 0 1 10.563 12zm6.024 4.142V7.86a4.442 4.442 0 0 1 2.85 4.14 4.443 4.443 0 0 1-2.85 4.142z"
            fill="#fff"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`BankCard Snapshots WithMoney 1`] = `
<DocumentFragment>
  <div
    class=""
    data-hovered="false"
    style="background-color: rgb(34, 34, 34); color: rgb(255, 255, 255);"
    tabindex="0"
  >
    <div
      class=""
    >
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="32"
          viewBox="0 0 32 32"
          width="32"
        >
          <path
            d="M24.048 4H7.993A3.99 3.99 0 0 0 4 7.986v16.028A3.99 3.99 0 0 0 7.993 28h16.013A3.99 3.99 0 0 0 28 24.014V7.986C28.041 5.786 26.252 4 24.048 4zm2.745 19.599c-.041.041-.125.083-.166.041a20.45 20.45 0 0 0-10.607-2.948A20.45 20.45 0 0 0 5.414 23.64c-.041.042-.125 0-.166-.041-.042-.083 0-.125.041-.166 2.829-2.16 5.241-4.9 7.113-8.139A27.644 27.644 0 0 0 15.896 5.08c0-.042.041-.125.124-.125.084 0 .125.042.125.125a27.642 27.642 0 0 0 3.494 10.214 27.38 27.38 0 0 0 7.113 8.097c.083.083.083.125.041.208z"
            fill="#FE5000"
          />
        </svg>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="16"
          viewBox="0 0 16 16"
          width="16"
        >
          <path
            clip-rule="evenodd"
            d="M6.5 2.5C6.5 1.67157 7.17157 1 8 1C8.82843 1 9.5 1.67157 9.5 2.5C9.5 3.32843 8.82843 4 8 4C7.17157 4 6.5 3.32843 6.5 2.5ZM6.5 8C6.5 7.17157 7.17157 6.5 8 6.5C8.82843 6.5 9.5 7.17157 9.5 8C9.5 8.82843 8.82843 9.5 8 9.5C7.17157 9.5 6.5 8.82843 6.5 8ZM8 12C7.17157 12 6.5 12.6716 6.5 13.5C6.5 14.3284 7.17157 15 8 15C8.82843 15 9.5 14.3284 9.5 13.5C9.5 12.6716 8.82843 12 8 12Z"
            fill="currentColor"
            fill-rule="evenodd"
          />
        </svg>
      </div>
    </div>
    <div
      class=""
    >
      <div
        class=""
      >
        <span
          data-family="false"
        >
          ** 4100
        </span>
        <h4
          data-family="false"
        >
          100 ₽
        </h4>
      </div>
      <div
        class=""
      >
        <svg
          aria-hidden="true"
          class="SvgIcon"
          fill="none"
          focusable="false"
          height="24"
          viewBox="0 0 36 24"
          width="36"
        >
          <path
            d="M19.181 21c4.872.024 9.319-4.012 9.319-8.922 0-5.369-4.447-9.08-9.319-9.078h-4.193C10.058 2.998 6 6.71 6 12.078c0 4.91 4.058 8.946 8.988 8.922h4.193z"
            fill="#0079BE"
          />
          <path
            d="M15.008 3.744c-4.505.002-8.156 3.689-8.157 8.24.001 4.55 3.652 8.237 8.157 8.239 4.507-.002 8.158-3.689 8.159-8.239-.001-4.551-3.652-8.238-8.159-8.24zm-5.17 8.24a5.23 5.23 0 0 1 3.32-4.874v9.747a5.227 5.227 0 0 1-3.32-4.873zm7.02 4.875V7.11a5.227 5.227 0 0 1 3.322 4.875 5.228 5.228 0 0 1-3.321 4.875z"
            fill="#fff"
          />
          <path
            d="M20.06 21c5.197.024 9.94-4.012 9.94-8.922C30 6.71 25.257 2.998 20.06 3h-4.472C10.328 2.998 6 6.71 6 12.078c0 4.91 4.329 8.946 9.588 8.922h4.472z"
            fill="#0079BE"
          />
          <path
            d="M15 5a7.001 7.001 0 0 0 0 14 7 7 0 0 0 0-14zm-4.437 7a4.444 4.444 0 0 1 2.848-4.14v8.28A4.442 4.442 0 0 1 10.563 12zm6.024 4.142V7.86a4.442 4.442 0 0 1 2.85 4.14 4.443 4.443 0 0 1-2.85 4.142z"
            fill="#fff"
          />
        </svg>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
